name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -403,13 +402,14 @@ protected boolean isIgnoreContentType()
     {
         try
         {
-            String escapedUrl = new URI(url).toASCIIString();
-            return parse(escapedUrl, Jsoup.connect(escapedUrl).userAgent(getUserAgent())
-                            .ignoreContentType(isIgnoreContentType()).timeout(30000).get(), errors);
+            URL urlObject = new URL(url);
+            String html = new WebAccess(urlObject.getHost(), urlObject.getPath()).withScheme(urlObject.getProtocol())
+                            .addUserAgent(getUserAgent()).get();
+            return parse("n/a", Jsoup.parse(html), errors); //$NON-NLS-1$
```

Comment:
Statt ```n/a``` gibt doch das URL#toString Objekt weiter. Das "url" Argument wird nur dazu genutzt um in (mögliche) Fehlermeldungen die URL zu schreiben und damit dem Benutzer es einfacher zu machen welche URL denn gerade wohl ein Problem hat.